lib/bolt_server/transport_app.rb

```diff
@@ -54,7 +54,13 @@ def initialize(config)
 
       body = JSON.parse(request.body.read)
       schema_error = JSON::Validator.fully_validate(@schemas["ssh-run_task"], body)
-      return [400, schema_error.join] if schema_error.any?
+      if schema_error.any?
+        msg = "There was an error validating the request body."
```

Comment:
This would make sense as a helper, to avoid having inconsistent responses in the future.